src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ManageContent/ManageContentPageServiceTests.cs

```diff
@@ -377,9 +374,7 @@ public async Task GetManageContentPageViewModel()
                 Assert.Null(contentPublication.ExternalMethodology);
                 Assert.Equal("test-publication", contentPublication.Slug);
                 Assert.Equal("Publication", contentPublication.Title);
-                Assert.Equal("Theme", contentPublication.Topic.Theme.Title);
-                Assert.NotNull(contentPublication.Topic);
-                Assert.NotNull(contentPublication.Topic.Theme);
+                Assert.NotNull(contentPublication);
```

Comment:
Should this be `contentPublication.Theme`? If it is null then the previous two asserts are going to fail before we reach this.

Reply:
There is no `Theme` on `contentPublication`, and the same null assertion is done further up so I'll just remove this line.